sql/ddl_api.sql

```diff
@@ -81,6 +84,22 @@ BEGIN
         USING ERRCODE = 'IO102';
     END IF;
 
+    -- We don't use INTO STRICT here because that error (no column) is surfaced
+    -- later. We only care if this is INTEGER, all other types (including no
+    -- type) can continue as before.
+    SELECT atttypid
+    INTO time_type
+    FROM pg_attribute
+    WHERE attrelid = main_table AND attname = time_column_name;
+
+    chunk_time_interval_actual := chunk_time_interval;
+    IF time_type = 'INTEGER'::REGTYPE AND chunk_time_interval = _timescaledb_internal.interval_to_usec('1 month') THEN
```

Comment:
So the proposal is to set it to NULL by default, and if it's a TIMESTAMP/TZ we set it to `1 month`, otherwise throw an error saying it needs to be supplied?